This fixes the CMake build. Fixed upstream here:
https://code.google.com/p/box2d/source/detail?r=313

This does not use the bundled glew and glfw.

Lubomir Rintel <lkundrak@v3.sk>

diff -urp Box2D-2.3.1/Box2D/CMakeLists.txt Box2D-2.3.1.fix/Box2D/CMakeLists.txt
--- Box2D-2.3.1/Box2D/CMakeLists.txt	2015-02-20 14:14:39.000000000 +0100
+++ Box2D-2.3.1.fix/Box2D/CMakeLists.txt	2015-02-20 14:31:34.347669244 +0100
@@ -26,8 +26,6 @@ if(BOX2D_BUILD_EXAMPLES)
 
   # Testbed and dependencies.
   find_package(OpenGL REQUIRED)
-  add_subdirectory(freeglut)
-  add_subdirectory(glui)
   add_subdirectory(Testbed)
 endif(BOX2D_BUILD_EXAMPLES)
 
diff -urp Box2D-2.3.1/Box2D/Testbed/CMakeLists.txt Box2D-2.3.1.fix/Box2D/Testbed/CMakeLists.txt
--- Box2D-2.3.1/Box2D/Testbed/CMakeLists.txt	2015-02-20 14:14:39.000000000 +0100
+++ Box2D-2.3.1.fix/Box2D/Testbed/CMakeLists.txt	2015-02-20 14:32:42.322404515 +0100
@@ -1,13 +1,17 @@
 # Some flags for Freeglut and GLUI.
-add_definitions( -DFREEGLUT_EXPORTS -DFREEGLUT_STATIC -D_CRT_SECURE_NO_WARNINGS )
+add_definitions( -D_CRT_SECURE_NO_WARNINGS )
 
 # Define the framework files.
 set(Testbed_Framework_SRCS
 	Framework/Main.cpp
-	Framework/Render.cpp
-	Framework/Render.h
+	Framework/RenderGL3.cpp
+	Framework/RenderGL3.h
 	Framework/Test.cpp
 	Framework/Test.h
+	Framework/DebugDraw.cpp
+	Framework/DebugDraw.h
+	Framework/imgui.cpp
+	Framework/imgui.h
 )
 
 #define the test files.
@@ -72,7 +76,7 @@ if(APPLE)
 	# We are not using the Apple's framework version, but X11's
 	include_directories( /usr/X11/include )
 	link_directories( /usr/X11/lib )
-	set (OPENGL_LIBRARIES GL GLU GLUT X11)
+	set (OPENGL_LIBRARIES GL GLU X11)
 elseif(WIN32)
 	set (ADDITIONAL_LIBRARIES winmm)
 endif(APPLE)
@@ -85,8 +89,8 @@ add_executable(Testbed
 target_link_libraries (
 	Testbed
 	Box2D
-	freeglut_static
-	glui
+	glfw
+	GLEW
 	${ADDITIONAL_LIBRARIES}
 	${OPENGL_LIBRARIES}
 ) 
diff --git a/Box2D/Testbed/Framework/Test.h b/Box2D/Testbed/Framework/Test.h
index 7e621d4..bb4cb13 100644
--- a/Box2D/Testbed/Framework/Test.h
+++ b/Box2D/Testbed/Framework/Test.h
@@ -25,7 +25,7 @@
 #if defined(__APPLE__)
 #include <OpenGL/gl3.h>
 #else
-#include <glew/glew.h>
+#include <GL/glew.h>
 #endif
 #include <glfw/glfw3.h>
 
diff --git a/Box2D/Testbed/Framework/DebugDraw.cpp b/Box2D/Testbed/Framework/DebugDraw.cpp
index 8299611..2a7da6b 100644
--- a/Box2D/Testbed/Framework/DebugDraw.cpp
+++ b/Box2D/Testbed/Framework/DebugDraw.cpp
@@ -21,10 +21,10 @@
 #if defined(__APPLE_CC__)
 #include <OpenGL/gl3.h>
 #else
-#include <glew/glew.h>
+#include <GL/glew.h>
 #endif
 
-#include <glfw/glfw3.h>
+#include <GLFW/glfw3.h>
 #include <stdio.h>
 #include <stdarg.h>
 
diff --git a/Box2D/Testbed/Framework/Main.cpp b/Box2D/Testbed/Framework/Main.cpp
index b494da9..79e2c05 100644
--- a/Box2D/Testbed/Framework/Main.cpp
+++ b/Box2D/Testbed/Framework/Main.cpp
@@ -24,10 +24,10 @@
 #if defined(__APPLE__)
 #include <OpenGL/gl3.h>
 #else
-#include <glew/glew.h>
+#include <GL/glew.h>
 #endif
 
-#include <glfw/glfw3.h>
+#include <GLFW/glfw3.h>
 #include <stdio.h>
 
 #ifdef _MSC_VER
diff --git a/Box2D/Testbed/Framework/RenderGL3.cpp b/Box2D/Testbed/Framework/RenderGL3.cpp
index 4613541..115cb93 100644
--- a/Box2D/Testbed/Framework/RenderGL3.cpp
+++ b/Box2D/Testbed/Framework/RenderGL3.cpp
@@ -25,7 +25,7 @@
 #ifdef __APPLE__
 #include <OpenGL/gl3.h>
 #else
-#include <glew/glew.h>
+#include <GL/glew.h>
 #include <GL/gl.h>
 #endif
 
diff --git a/Box2D/Testbed/Framework/Test.h b/Box2D/Testbed/Framework/Test.h
index bb4cb13..f6c69f3 100644
--- a/Box2D/Testbed/Framework/Test.h
+++ b/Box2D/Testbed/Framework/Test.h
@@ -27,7 +27,7 @@
 #else
 #include <GL/glew.h>
 #endif
-#include <glfw/glfw3.h>
+#include <GLFW/glfw3.h>
 
 #include <stdlib.h>
 
